CorsixTH/Lua/dialogs/resizables/options.lua

```diff
@@ -372,6 +377,12 @@ function UIOptions:buttonHotkey()
   self.ui:addWindow(window)
 end
 
+function UIOptions:buttonJukebox()
+  if self.ui.app.config.audio and self.ui.app:isAudioEnabled() then
+    self.ui:addWindow(UIJukebox(self.ui.app))
```

Comment:
`UIOptions` is aware of `app` without going through `ui`. But maybe your path is better,